lib/stores/file_tree.js

```diff
@@ -16,12 +16,14 @@ const writeFile = promisify(fs.writeFile);
 class FileTree {
   constructor (options) {
     this._dir = path.resolve(options.path);
+    this._timeout = options.lock_timeout_ms ?? 200;
```

Comment:
This is the length of time **another thread** will wait for the lock to be released and thus, an upper limit on how long an HTTP request can take before it gives up because of the lock. 

Firefox will wait 10 minutes for an HTTP request to succeed, other browsers wait longer, giving us plenty of headroom there.

Defaults for the remoteStorage library are:
  requestTimeout: 30000,
  syncInterval: 10000

I recommend we set the default value of `this._timeout` to 30_000 (30 seconds)